astropy/utils/data.py

```diff
@@ -1015,7 +1019,7 @@ def _download_file_from_source(source_url, show_progress=True, timeout=None,
 
 
 def download_file(remote_url, cache=False, show_progress=True, timeout=None,
-                  sources=None, pkgname='astropy'):
+                  sources=None, pkgname='astropy', http_headers={}):
```

Comment:
Preferable to use None here rather than a mutable default argument; it's a customary way to avoid that pitfall.